src/ui/LeftSearchPanel.java

```diff
@@ -0,0 +1,110 @@
+package ui;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+
+
+/**
+ * This is the panel that all the images will be loaded in a small icons list
+ */
+public class LeftSearchPanel extends JPanel {
+
+    private JToolBar jToolBar;
+    private ImageIcon iconPath;
+    private JButton chooseFileButton;
+    private File selectedFile;
+    private JList imagesList;
+    private ImageListModel imageListModel;
+    private JScrollPane scrollPaneJlist;
+    private RightPanelViewer rightPanelViewer;
+
+    /**
+     * constructor load all static metods and apply  a Border layout
```

Comment:
misspelling metods, methods? and the documentation does not help to understand what it does to the UI